src/main/kotlin/carracing/CarRacingGameManager.kt

```diff
@@ -0,0 +1,29 @@
+package carracing
+
+fun main() {
+    val inputView = InputView()
+    val outputView = OutputView(OutputViewResolver())
+    val numberGenerator = RandomNumberGenerator()
+
+    val carRacingGameManager = CarRacingGameManager(inputView, outputView, numberGenerator)
+    carRacingGameManager.gameStart()
+}
+
+class CarRacingGameManager(
+    private val inputView: InputView,
+    private val outputView: OutputView,
+    private val numberGenerator: NumberGenerator,
+) {
+    fun gameStart() {
+        val totalCar = inputView.inputTotalCar()
+        val totalRound = inputView.inputTotalRound()
+        val minConditionToMove = 4
+
+        val cars = Cars.create(totalCar, 0, minConditionToMove)
```

Comment:
If you're not going to receive minConditionToMove as input, I think it would be better to move it to another object.
> I think you can use const :)